internal/helm/flags/flag.go

```diff
@@ -49,6 +50,11 @@ func (f *Flags) AddTo(flagSet *pflag.FlagSet) {
 		":8080",
 		"The address the metric endpoint binds to",
 	)
+	flagSet.StringVar(&f.ProbeAddr,
+		"health-probe-bind-address",
+		":8081",
```

Comment:
Port number needs to stay the same too
```suggestion
		":6789",
```

Reply:
Here we have the Helm flags. It is new for the helm. So, we can keep the same port used by Go. Note that for the release 1.0 we agree in try to keep to use the same ports and flags in order to be easier we give support and keep it maintained as for the users since is not required remember and memorize that each type language is using a diff port by default. see: https://github.com/operator-framework/operator-sdk/issues/3358 

So, for 2.0 we might be able to keep all using the same.

Reply:
I think you're right, I thought I saw helm-operator using 6789 but this is a net-new feature.